# Generated by Django 4.1.7 on 2023-04-12 10:16

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):
    dependencies = [
        (
            "consultants",
            "0006_alter_consultants_options_alter_consultants_managers_and_more",
        ),
    ]

    operations = [
        migrations.AlterField(
            model_name="consultants",
            name="username",
            field=models.CharField(
                error_messages={"unique": "A user with that username already exists."},
                help_text="Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.",
                max_length=150,
                unique=True,
                validators=[django.contrib.auth.validators.UnicodeUsernameValidator()],
                verbose_name="username",
            ),
        ),
    ]
